app/models/member_query_object.rb

```diff
@@ -4,94 +4,119 @@ class MemberQueryObject
   def self.all
     self.uri_builder('
         PREFIX parl: <http://id.ukpds.org/schema/>
-      CONSTRUCT{
-         ?seatIncumbency
-           a parl:SeatIncumbency ;
-           parl:seatIncumbencyHasMember ?member ;
-           parl:seatIncumbencyHasHouseSeat ?houseSeat .
-         ?member
-           a parl:Person ;
-           parl:personGivenName ?givenName ;
-           parl:personFamilyName ?familyName ;
-           parl:partyMemberHasPartyMembership ?partyMembership .
-         ?partyMembership
-           a parl:PartyMembership ;
-           parl:partyMembershipHasParty ?party .
-         ?party
-           a parl:Party ;
-           parl:partyName ?partyName .
+      CONSTRUCT {
          ?houseSeat
-           a parl:HouseSeat ;
-           parl:houseSeatHasHouse ?house ;
-           parl:houseSeatHasConstituencyGroup ?constituencyGroup .
-         ?house
-           a parl:House ;
-           parl:houseName ?houseName .
-         ?constituencyGroup
-           a parl:ConstituencyGroup ;
-           parl:constituencyGroupName ?constituencyGroupName .
+         a parl:HouseSeat ;
+         parl:houseSeatHasHouse ?house ;
+         parl:houseSeatHasConstituencyGroup ?constituencyGroup .
+        ?person
+         a parl:Person ;
+         parl:personGivenName ?givenName ;
+         parl:personFamilyName ?familyName ;
+         parl:memberHasIncumbency ?incumbency ;
```

Comment:
Change ?incumbency to ?seatIncumbency - as in the view we only need seatIncumbency